.changeset/brave-items-invite.md

```diff
@@ -0,0 +1,23 @@
+---
+'braid-design-system': minor
+---
+
+---
+new:
+  - Playroom
+---
```

Comment:
The additional front matter is not required for playroom as its not a documented entry point.
```suggestion
```